org.libretro.RetroArch.json

```diff
@@ -59,6 +60,29 @@
               "sha256": "0f4620e11fa283fedafb474427c8e96bf149511a1804bdc47350963ae5cf54d8"
             }
           ]
+        },
+        {
+          "name": "nvidia-cg-tookit",
+          "sources": [
+          {
+              "type": "archive",
+              "url": "http://developer.download.nvidia.com/cg/Cg_3.1/Cg-3.1_April2012_x86_64.tgz",
+              "sha256": "e8ff01e6cc38d1b3fd56a083f5860737dbd2f319a39037528fb1a74a89ae9878"
+          },
+            {
+              "type": "script",
```

Comment:
It is described in `flatpak-manifest` man page. I don't think its up-to-date version is online anywhere. You can look at other Flathub packages like [gnucash](https://github.com/flathub/org.gnucash.GnuCash/blob/master/org.gnucash.GnuCash.json#L75) for examples.